package.json

```diff
@@ -1,12 +1,28 @@
 {
-  "author": "MaibornWolff GmbH <dist_intern_codecharta@maibornwolff.de>",
-  "homepage": "https://maibornwolff.github.io/codecharta/",
-  "bugs": {
-    "url": "https://github.com/MaibornWolff/codecharta/issues",
-    "email": "dist_intern_codecharta@maibornwolff.de"
-  },
-  "name": "codecharta",
-  "version": "0.0.2",
-  "description": "MaibornWolff CodeCharta placeholder",
-  "license": "BSD-3-Clause"
+	"author": "MaibornWolff GmbH <dist_intern_codecharta@maibornwolff.de>",
+	"homepage": "https://maibornwolff.github.io/codecharta/",
+	"bugs": {
+		"url": "https://github.com/MaibornWolff/codecharta/issues",
+		"email": "dist_intern_codecharta@maibornwolff.de"
+	},
+	"name": "codecharta",
+	"version": "0.0.2",
+	"description": "MaibornWolff CodeCharta placeholder",
+	"license": "BSD-3-Clause",
+	"scripts": {
+		"format": "prettier --write \"./**/*\"",
+		"format:quick": "pretty-quick --staged",
+		"lint:visualization": "cd visualization && npm run lint:fix"
+	},
+	"husky": {
+		"hooks": {
+			"pre-commit": "npm run lint:visualization && npm run format:quick",
```

Comment:
Running the formatter during the commit stage changes files locally without adding the changes to the commit. That might be confusing? I personally use [lint-staged](https://github.com/okonet/lint-staged) due to that reason. Also check https://prettier.io/docs/en/precommit.html#:~:text=You%20can%20use%20Prettier%20with,git%20add%20before%20you%20commit. for further options.

Reply:
That's why we use `pretty-quick`. It allows us to add the formatted files again before commiting. 
As long as we run lint before commit, everything should be staged.

https://www.npmjs.com/package/pretty-quick#--staged-only-git

Reply:
Perfect!